Update the description of a cookbook (#3339)
chef/chef-web-docs: themes/docs-new/layouts/shortcodes/cookbooks_summary.md

A cookbook is the fundamental unit of configuration and policy distribution in Chef Infra.

A cookbook defines a scenario and contains everything that is required to support that scenario:

> - Recipes that specify Chef Infra built-in resources to use and the order in which they are to be applied

Reviewer: ```suggestion
- Recipes that specify which Chef Infra built-in resources to use, as well as the order in which they are to be applied
```

> - Attribute values allowing different configuratinos to be applied based on environments such as dev vs. production.

Reviewer: ```suggestion
- Attribute values, which allow environment-based configurations such as `dev` or `prodution`.
```

> - Custom Resources used to extend Chef Infra beyond the built-in resources.

Reviewer: ```suggestion
- Custom Resources for extending Chef Infra beyond the built-in resources.
```

> - Files and Templates for distribution to systems.

Reviewer: For distributing what? I'm putting "information" but we should clarify this.

```suggestion
- Files and Templates for distributing information to systems.
```

> - Custom Ohai Plugins for extending system configuration collection.

Reviewer: ```suggestion
- Custom Ohai Plugins for extending system configuration collection beyond the Ohai defaults.
```

> - The `metadata.rb` file describing the cookbook itself and any dependendencies it may have.

Reviewer: ```suggestion
- The `metadata.rb` file, which describes the cookbook itself and any dependencies it may have.
```

Author: Do you not like my creative way of spelling that?
